pyfixest/estimation/feiv_.py

```diff
@@ -133,6 +139,28 @@ def get_fit(self) -> None:
         _Z = self._Z
         _Y = self._Y
 
+        #  Start First Stage
+
+        model1 = Feols(
```

Comment:
I think it correct that the endogenous variable is always the first one in `_X`. But it would likely be safer to use the [endogvar](https://github.com/py-econometrics/pyfixest/blob/b30512d63fc7186e0d5b6058b094fd0a0b7d6cc1/pyfixest/estimation/FixestMulti_.py#L232) created via the `model_matrix_fixest` function? 

This would require to add an `endogvar` argument to `Feiv.__init__` and to pass it to `Feiv.get_fit()`. 

We would also have to demean it in FixestMulti.

Reply:
I think your practice is more sensible as it encompasses more general cases. I will take this into account in the next iteration!